packages/tabsheet/src/vaadin-tabsheet.js

```diff
@@ -157,7 +157,7 @@ class TabSheet extends ControllerMixin(DelegateStateMixin(ElementMixin(ThemableM
 
   /** @override */
   static get delegateProps() {
-    return ['selected'];
+    return ['selected', 'theme'];
```

Comment:
**nit:** It should probably delegate `theme` as the attribute because setting theme via property has been deprecated.

Reply:
Done